### PR TITLE
Document roadmap and dev process

### PR DESCRIPTION
**Pull request by leandrocp**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,25 +20,11 @@ Pull requests correcting spelling or grammar mistakes are always welcome.
...
-Please try to answer at least the following questions when reporting a bug:
-
- - Which version of the project did you use when you noticed the bug?
- - How do you reproduce the error condition?
- - What happened that you think is a bug?
- - What should it do instead?
-
-It would really help the maintainers if you could provide a reduced test case
-that reproduces the error condition.
+Please open a [bug reporet](https://github.com/BeaconCMS/beacon/issues/new?template=bug_report.md&labes=bug).
```
Comment (brzpegasus) [inline CONTRIBUTING.md:23]: s/reporet/report/

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -45,3 +36,14 @@ get accepted:
 
 If your pull-request addresses an issue then please add the corresponding
 issue's number to the description of your pull-request.
+
+## Prioritization
+
+Issues tagged as [roadmap](https://github.com/BeaconCMS/beacon/labels/roadmap) indicate the work that is planned to be worked
+next on the short term, those have more priority over the other issues and will be included in the next releases.
```
Comment (brzpegasus) [inline CONTRIBUTING.md:43]: Start a new sentence at "those have more priority".

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -45,3 +36,14 @@ get accepted:
...
+
+Engaging on [feature request issues](https://github.com/BeaconCMS/beacon/issues?q=is%3Aissue+is%3Aopen+label%3Afeature-request) also helps us to prioritize the work,
+so feel free to add comments or discusson on feature request you'd like to see implemented and those will have more priority over the other issues.
```
Comment (brzpegasus) [inline CONTRIBUTING.md:46]: s/discusson/discussion/
s/on feature request/on feature requests/

File: RELEASE.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+# Release Process
+
+This policy applies to Beacon, BeaconLiveAdmin, and any other project in the BeaconCMS organization.
+
+## Guidelines
+
+1. Follow [Semantic Versioning](https://semver.org/)
+
+   * For projects on v0.x releases, breaking changes are released as minor versions., for example in v0.2.0
```
Comment (brzpegasus) [inline RELEASE.md:9]: Remove the period before the comma (here and in the next bullet point).

File: RELEASE.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+   * For projects on v1.x releases, breaking changes are released as major versions., for example in v2.0.0
+
+2. Each project has its own lifecycle and releases, one may evolve faster than others
```
Comment (brzpegasus) [inline RELEASE.md:12]: Missing period at the end of this sentence.

File: RELEASE.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+3. A new major or minor version is released once a month containing features, fixes, and potentially breaking changes.
+
+   * The type of the version, either major or minor is decided based on the changes included in that release and the current version.
+   * We might skip releasing a new version if there's no changes in that period of time.
```
Comment (brzpegasus) [inline RELEASE.md:17]: "if there's no changes" -> "if there are no changes"

File: RELEASE.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+5. Bug fixes and security patches are released as soon as possible, either as a patch or minor version depending on the current version.
+
+## Git strategy
+
+Change are applied to the `main` branch through feature branches. A pull requested is opened, reviewed, and merged into `main`
```
Comment (brzpegasus) [inline RELEASE.md:23]: s/Change/Changes/
s/A pull requested/A pull request/

File: RELEASE.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+
+## Release steps
+
+1. Checkout the `main` branch and make sure it's up to date with upstream
+2. Run `mix assets.build`
+3. Update the `CHANGELOG.md` file to list all changes included in the release and add the current date
+4. Commit the changes to upstream `main` if necessary
+5. Checkout the release branch and make sure it's up to date with upstream
+6. Make sure all the relevant changes have been cherry-picked from `main` to the release branch
+7. Update the version in the files `mix.exs` and `package.json` and throut docs/ files
```
Comment (brzpegasus) [inline RELEASE.md:40]: "and throut" -> Did you mean "and through" or "and throughout"?

File: RELEASE.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+8. Run `mix assets.build`
+9. Make sure all relevant changes, especially breaking changes, are documented in a upgrade guide in `guides/upgrading`
```
Comment (brzpegasus) [inline RELEASE.md:42]: "in a upgrade guide" -> "in an upgrade guide"

File: ROADMAP.md [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+- Provide more guides and recipes as we learn more about what the community needs
+- Make code documentation part of the development process
+- Update the content of https://beaconcms.org
+- Add support for more Phoenix and LiveView resources such as JS hooks
+- Admin interactive tour for user onboarding
+
+## Content Creation Experience
+
+#### Visual Editor
+- High-level visual controls that generate the underlyng styles, effects, and attributes in general. Including bot not limited to:
```
Comment (brzpegasus) [inline ROADMAP.md:24]: s/underlyng/underlying/
s/bot not/but not/

File: ROADMAP.md [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
+  - colors
+  - sizing
+  - positioning
+  - shadows
+  - visibility
+  - navigation (links)
+  - eex (:if conditionals for example)
+  - animiations
```
Comment (brzpegasus) [inline ROADMAP.md:32]: s/animiations/animations/

File: ROADMAP.md [hunk 12]
```diff
@@ -0,0 +1,88 @@
...
+
+#### Code Editor
+- Rich Markdown editor that provides high-level visual controls to upload media assets, create page links, embed Phoenix components, and more.
+
+#### Media Library
+- Redesign Media Library - https://www.figma.com/design/k8NhxbRRb8hm4fc1RQHms1/Media-Library?node-id=30-57434
+
+#### Admin Interface
+- Adopt a component/UI library to improve the standard look and feel of admin pages
+- Expose built-in components and colors to allow users build custom pages easily
```
Comment (brzpegasus) [inline ROADMAP.md:46]: "allow users build" -> "allow users to build"

**Review by brzpegasus — CHANGES_REQUESTED** [review]

**Review by brzpegasus — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 13]
```diff
@@ -0,0 +1,25 @@
...
+---
+
+### Environment
+
+* Elixir version (elixir -v):
+* Phoenix version (mix deps):
+* Phoenix LiveView version (mix deps):
+* Beacon version (mix deps):
+* Operating system:
+* Browsers you attempted to reproduce this bug on (the more the merrier):
```
Comment (APB9785) [inline .github/ISSUE_TEMPLATE/bug_report.md:17]:
```suggestion
* Browsers on which you attempted to reproduce this bug (the more the merrier):
```

File: CONTRIBUTING.md [hunk 14]
```diff
@@ -1,8 +1,13 @@
...
+# Contributing to Beacon
 
-## Issue Tags
+## Guidelines
 
-The tag [good first issue](https://github.com/BeaconCMS/beacon/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) is recommended for fist time contributors, they are usually small fixes or improvements that don't require too much knowledge of the project. The tag [help wanted](https://github.com/BeaconCMS/beacon/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22+-label%3A%22good+first+issue%22+) represents issues that would benefit from contributors but may need more knowledge or discussion over the requirements.
+* Issues aligned with our [roadmap](https://github.com/BeaconCMS/beacon_live_admin/blob/main/ROADMAP.md) have higher priority
+  and those are [labeled as roadmap](https://github.com/BeaconCMS/beacon/labels/roadmap)
+* [Bugs](https://github.com/BeaconCMS/beacon/labels/bug) tend to have higher priority unless they are of minor impact
+* If unsure where to start, look for issues tagged as [good first issue](https://github.com/BeaconCMS/beacon/labels/good%20first%20issue) to find good candidates of issues to start contributing
```
Comment (APB9785) [inline CONTRIBUTING.md:8]:
```suggestion
* If unsure where to start, look for issues tagged as [good first issue](https://github.com/BeaconCMS/beacon/labels/good%20first%20issue) to find good candidates for your first contribution
```

File: CONTRIBUTING.md [hunk 15]
```diff
@@ -1,8 +1,13 @@
...
+* Issues tagged [help wanted](https://github.com/BeaconCMS/beacon/labels/help%20wanted) are also good candidates but may require more knowledge of the project and time
```
Comment (APB9785) [inline CONTRIBUTING.md:9]:
```suggestion
* Issues tagged [help wanted](https://github.com/BeaconCMS/beacon/labels/help%20wanted) are also good candidates but may require more project knowledge and/or time investment
```

File: CONTRIBUTING.md [hunk 16]
```diff
@@ -45,3 +36,14 @@ get accepted:
 
 If your pull-request addresses an issue then please add the corresponding
 issue's number to the description of your pull-request.
+
+## Prioritization
+
+Issues tagged as [roadmap](https://github.com/BeaconCMS/beacon/labels/roadmap) indicate the work that is planned to be worked
+next on the short term. Those have more priority over the other issues and will be included in the next releases.
```
Comment (APB9785) [inline CONTRIBUTING.md:43]:
```suggestion
Issues tagged as [roadmap](https://github.com/BeaconCMS/beacon/labels/roadmap) indicate the work that is planned to be done
next in the short term. Those have more priority over the other issues and will be included in the next releases.
```

File: CONTRIBUTING.md [hunk 17]
```diff
@@ -45,3 +36,14 @@ get accepted:
...
+
+Engaging on [feature request issues](https://github.com/BeaconCMS/beacon/issues?q=is%3Aissue+is%3Aopen+label%3Afeature-request) also helps us to prioritize the work,
+so feel free to add comments or discuss on feature requests you'd like to see implemented and those will have more priority over the other issues.
+
+Finally keep in mind that we can't guarantee which exact issues will be included in the very next release and we may give more
+priority to specific issues as decided by the core team as needed.
```
Comment (APB9785) [inline CONTRIBUTING.md:49]:
```suggestion
Finally keep in mind that we can't guarantee which exact issues will be included in the very next release; the core team may
give more priority to specific issues as needed.
```

File: RELEASE.md [hunk 18]
```diff
@@ -0,0 +1,46 @@
...
+3. A new major or minor version is released once a month containing features, fixes, and potentially breaking changes.
+
+   * The type of the version, either major or minor is decided based on the changes included in that release and the current version.
+   * We might skip releasing a new version if there are no changes in that period of time.
+
+5. Bug fixes and security patches are released as soon as possible, either as a patch or minor version depending on the current version.
+
+## Git strategy
+
+Changes are applied to the `main` branch through feature branches. A pull requeste is opened, reviewed, and merged into `main`
```
Comment (APB9785) [inline RELEASE.md:23]:
```suggestion
Changes are applied to the `main` branch through feature branches. A pull request is opened, reviewed, and merged into `main`
```

**Review by APB9785 — APPROVED** [review]

**Pull request merged**